packages/wallets-helpers/src/walletconnect/getWalletConnectConnector.ts

```diff
@@ -14,7 +14,7 @@ export const getWalletConnectConnector = ({
   chains: Chain[];
 }) => {
   if (!projectId) {
-    throw new Error(
+    console.warn(
```

Comment:
what should be returned in that case?

Reply:
@Jabher WalletConnect controller, as before. It will not work properly without the id, but there is no way to quickly fix this situation right now, because of wagmi requirements and at least it will not break the build